seaseq-control.wdl

```diff
@@ -654,283 +661,15 @@ workflow seaseq {
     #   Complete Summary statistics
 
     #collate correct files for downstream analysis
+    # s_mergebam_afterbklist has no BAM index
+    # s_mapping.bklist_bam has s_mapping.bklist_index
+    # s_mapping.sorted_bam has s_mapping.bam_index
     File sample_bam = select_first([s_mergebam_afterbklist, s_mapping.bklist_bam, s_mapping.sorted_bam])
+    File sample_bai = select_first([s_mergebam_afterbklist_index, s_mapping.bklist_index, s_mapping.bam_index])
     File control_bam = select_first([c_mergebam_afterbklist, c_mapping.bklist_bam, c_mapping.sorted_bam])
+    File control_bai = select_first([c_mergebam_afterbklist_index, c_mapping.bklist_index, c_mapping.bam_index])
```

Comment:
will need the additional files for BAMtoGFF
s_rmdup_bam = select_first([s_merge_markdup.mkdupbam, s_mapping.mkdup_bam])
s_rmdup_bai = select_first([s_merge_mkdup.indexbam, s_mapping.mkdup_index])
c_rmdup_bam = select_first([c_merge_markdup.mkdupbam, c_mapping.mkdup_bam])
c_rmdup_bai = select_first([c_merge_mkdup.indexbam, c_mapping.mkdup_index])